src/app/CommandInputField.tsx

```diff
@@ -85,6 +93,16 @@ export const CommandInputField = observer((props: { controller: BottomPanelContr
       const selected = controller.selected;
       const autoCompletionValues = controller.autoCompletionValues;
 
+      if (selected === null && autoCompletionValues.length <= 0 && lastCmd.length > 0) {
```

Comment:
Refactor it using `else if` to simply the code

Reply:
Updated, thank you.